src/Core/InstanceConfigTrait.php

```diff
@@ -139,7 +212,7 @@ public function configShallow($key, $value = null)
     }
 
     /**
-     * Read a config variable
+     * Reads a config key.
```

Comment:
This is somewhat misleading as the method allows to read the whole config by providing `null```.